addons/website_hr_recruitment/views/website_hr_recruitment_templates.xml

```diff
@@ -285,10 +285,15 @@
                         <div t-if="job.address_id" class="d-flex flex-column align-items-baseline">
                             <span class="text-muted small">Location</span>
                             <h6 t-field="job.address_id" t-options='{
+                                "widget": "contact",
                                 "fields": ["city"],
                                 "no_tag_br": True
                             }'/>
                         </div>
+                        <div t-if="not job.address_id" class="d-flex flex-column align-items-baseline">
+                            <span class="text-muted small">Location</span>
+                            <h6>Remote</h6>
```

Comment:
I thought that you had to have an address or be remote (eg. if there is only one office they could not bother to fill the address if it was a known address or given elsewhere on the website).  But from the code it seems to at least bee the case, eg. the is_remote search search if there is no address.